crates/sui-types/src/execution_status.rs

```diff
@@ -182,6 +182,12 @@ pub enum ExecutionFailureStatus {
         Please run the Sui Move Verifier for more information."
     )]
     SuiMoveVerificationTimedout,
+
+    #[error(
+        "Shared object operations such a wrapping, freezing, or converting to owned are not \
+    allowed."
+    )]
+    SharedObjectOperationNotAllowed,
```

Comment:
When do we use `ExecutionFailureStatus::SharedObjectOperationNotAllowed` vs `ExecutionFailureStatus::CommandArgumentError { arg_idx: _, kind: CommandArgumentError::SharedObjectOperationNotAllowed }`?

Reply:
Maybe we should specify that this one is only from wrapping? (as far as I can tell at any rate)

Reply:
This error is only used in the case where we find that a shared object has been wrapped (does not occur in deleted, written, or transfers). This error occurs at the end of execution outside of the context of a command when computing final runtime results.

I will rename this to `SharedObjectWrapped` since this error only occurs in that case.